configuration/config-txt/video.md

```diff
@@ -58,17 +21,25 @@ overscan_top=24
 overscan_bottom=24
 ```
 
+### hdmi_force_hotplug
+
+Setting `hdmi_force_hotplug` to `1` pretends that the HDMI hotplug signal is asserted, so it appears that an HDMI display is attached. In other words, HDMI output mode will be used, even if no HDMI monitor is detected.
+
```

Comment:
Not keen on 'pretend' - 'emulates' perhaps?

Reply:
(Yes, I know this is what was there before!)

Reply:
Or "Setting `hdmi_force_hotplug` to `1` overrides the HDMI hotplug signal, so it appears that an HDMI display is always attached." ? :man_shrugging:

Reply:
Per the style guide, "pretend" is probably OK, since it fits with the intended chatty style, although I've never been entirely convinced that style is ideal for technical documentation. The parameters have slightly unfortunate names, but we are where we are. I've changed the wording and simplified the explanation.

Reply:
@lurch's suggested wording avoids "pretend" and also seems clearer, to me at least.